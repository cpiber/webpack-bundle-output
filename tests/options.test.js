const { webpack, file } = require('./shared');
const { getFixturePathSync } = require('jest-fixtures');
const { existsSync } = require('fs');
const { join } = require('path');

test('obeys output', async () => {
  const { out } = await webpack({
    entry: getFixturePathSync(__dirname, 'basic'),
  }, { output: 'test.json' });
  expect(existsSync(join(out, 'test.json'))).toBeTruthy();
});

test('obeys cwd', async () => {
  const { map } = await webpack({
    entry: getFixturePathSync(__dirname, 'basic'),
  }, { cwd: getFixturePathSync(__dirname) });
  expect(file(map)).toMatchSnapshot();
});