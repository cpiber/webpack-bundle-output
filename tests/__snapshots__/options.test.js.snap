// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`obeys cwd 1`] = `
{
  "basic/index.js": [
    "../out/0c059a296e7692b0b3e2c83306c9271d/main.js",
  ],
}
`;
